README.md

```diff
@@ -1,8 +1,7 @@
 # Introduction
 
 Welcome to the start of a new adventure! In this repository we will show you everything you need to do to setup a Minecraft Java Edition server!
-We will go through the steps in as much detail as possible
-Who knows? You might become the next Hypixel!
+We will go through the steps in as much detail as possible, and who knows? You might become the next Hypixel!
```

Comment:
```suggestion
We will go through the steps in as much detail as possible and who knows? You might become the next Hypixel!
```

Reply:
i just realised theres the option to commit suggestion lol